src/app/api/coupon/getCouponForServer.ts

```
@@ -0,0 +1,59 @@
+  let couponImageUrlList: string[] = [];
+
+  if (
+    // 보유한 쿠폰 코드가 있는 경우에만 DB에서 이미지 URL을 가져옴
+    couponCodeList &&
+    couponCodeList.coupons &&
+    couponCodeList.coupons.length > 0
+  ) {
+    const { coupons } = couponCodeList;
+
+    const { data: imageUrlListFromDB } = await supabase
+      .from('coupons')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)